pkg/lib/index.d.ts

```diff
@@ -0,0 +1,3 @@
+interface Window {
+    debugging?: string;
+}
```

Comment:
index? Is like global scope or?

Reply:
Ya.  It's so we can hit `window.debugging` from `ts` without warnings.